.github/workflows/build-windows-installer.yml

```diff
@@ -190,33 +220,27 @@ jobs:
         cd .\build_scripts
         .\build_windows-2-installer.ps1
 
-    - name: Upload Windows exe's to artifacts
-      uses: actions/upload-artifact@v3
-      with:
-        name: Windows-Exe
-        path: ${{ github.workspace }}\chia-blockchain-gui\Chia-win32-x64\
-
     - name: Upload Installer to artifacts
       uses: actions/upload-artifact@v3
       with:
         name: chia-installers-windows-exe-intel
         path: ${{ github.workspace }}\chia-blockchain-gui\release-builds\
 
     - name: Install AWS CLI
-      if: steps.check_secrets.outputs.HAS_AWS_SECRET
+      if: env.FULL_RELEASE == 'true' && steps.check_secrets.outputs.HAS_AWS_SECRET
       run: |
           msiexec.exe /i https://awscli.amazonaws.com/AWSCLIV2.msi
 
     - name: Configure AWS Credentials
-      if: steps.check_secrets.outputs.HAS_AWS_SECRET
+      if: env.FULL_RELEASE == 'true' && steps.check_secrets.outputs.HAS_AWS_SECRET
```

Comment:
This step should not just be on full release - we also need this to upload dev installers - should just be checking if HAS_AWS_SECRET like before

Reply:
fixed